modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -5,255 +5,258 @@
+== Review Third-Party Apps
 
-With those steps completed, stop your webserver.
+Review any installed third-party apps for compatibility with the new ownCloud release.
+Ensure that they are all disabled before beginning the upgrade. Third party apps are all apps that are not distributed by https://marketplace.owncloud.com/publishers/owncloud
 
+. Disable via Command Line
++
 [source,console]
 ----
-sudo service apache2 stop
+# This command lists all apps by <app-id> and app version
+sudo -u www-data php occ app:list
+
+# This command disables the app with the given <app-id>
+sudo -u www-data php occ app:disable <app-id>
 ----
 
-[[download-the-latest-installation]]
 == Download the Latest Installation
 
-Download the latest ownCloud server release from https://owncloud.org/install/[owncloud.org/install/] into an empty directory *outside* of your current installation.
-
-[IMPORTANT]
-====
-Once you have done that, ensure that the downloaded archive xref:maintenance/update.adoc#setting-permissions-for-updating[is owned by the web user and that the correct permissions are set].
-====
+Download the latest ownCloud server release from https://owncloud.org/install[owncloud.org/install] into an empty directory *outside* of your current installation.
+[source, console]
+----
+wget https://download.owncloud.org/community/owncloud-10.1.1.tar.bz2
+----
 
 NOTE: Enterprise users must download their new ownCloud archives from their accounts on
 https://customer.owncloud.com/owncloud/.
 
-[[setup-the-new-installation]]
-== Setup the New Installation
+To download the tarball from https://customer.owncloud.com
+you will need a command like this:
 
-Not all installations are the same, so we encourage you to take one of
-two paths to upgrade your ownCloud installation. These are
-the standard upgrade and the power user upgrade.
+[source,console]
+----
+wget https://username:password@customer.owncloud.com/link-to-tarball
+----
 
-If you’re reasonably new to ownCloud, or not too familiar with upgrading
-an ownCloud installation, please follow the standard upgrade. Otherwise,
-take the approach that you’re most comfortable with, likely the power
-user upgrade.
+== Upgrade
 
-NOTE: Regardless of which approach that you take, they will both assume that your existing ownCloud installation is located in the default location:
-`/var/www/owncloud`.
+NOTE: For this description we assume that your existing ownCloud installation is located in the
+default location: `/var/www/owncloud`. The path might differ, depending on your installation.
 
-[[the-standard-upgrade]]
-=== The Standard Upgrade
+=== Extract the New Source
 
-Delete all files and folders in your existing ownCloud directory
-(`/var/www/owncloud`) — *_except_* `data` and `config`.
+Rename your current ownCloud directory, for example, from `owncloud` to `owncloud-old-owncloud-version` and create a new ownCloud folder with:
+[source, console]
+----
+mv owncloud owncloud-old-owncloud-version
+mkdir owncloud
```

Comment:
This step is not needed. Later, you extract the tarball in /var/www/ which will create the folder "owncloud"